test/e2e/email-notifications/glad-alerts-emails.spec.ts

```diff
@@ -191,17 +194,19 @@ describe('GLAD-ALL emails (existing "glad-alerts" subscriptions should now use "
     });
 
     it('GLAD alert emails for admin 0 subscriptions work as expected', async () => {
+        const country = 'BRA'
+        const areaId = getUUID()
         EmailHelpersService.updateMonthTranslations();
         moment.locale('en');
         const subscriptionOne = await new Subscription(createSubscriptionContent(
             USERS.USER.id,
             'glad-alerts',
-            { params: { iso: { country: 'BRA' } } },
+            { params: { iso: { country }, area: areaId} },
         )).save();
 
         const { beginDate, endDate } = bootstrapEmailNotificationTests();
-        createMockGeostore('/v2/geostore/admin/BRA');
-
+        createMockGeostore(`/geostore/admin/${country}`, config.get('dataApi.url'));
```

Comment:
Good catch removing the `v2`! 👁️

Reply:
That's actually thanks to nock check that runs  for each test file ensuring every api call that's mocked is called with the correct arguments and raising an error if not. So all the existing tests (205 of them total) cover the updates I have made.